lib/player.rb

```diff
@@ -3,8 +3,8 @@ class Player
   attr_reader :piece, :name
   attr_accessor :winner
   
-  def initialize
-    @piece = "x"
+  def initialize(piece)
+    @piece = piece
```

Comment:
oh interesing. I was going to suggest a class method called "get_player_piece" in order for us to allow the player to choose the piece similar to the name, but this is fine!